app/assets/stylesheets/_style.css.scss

```diff
@@ -66,17 +69,15 @@ nav {
   }
 
   .user {
-    $_icon_size: 36px;
-
     .icon {
       display: inline;
 
       img {
-        margin-top: ($_nav_height - $_icon_size) / 2;
+        margin-top: ($nav_height - $icon_size) / 2;
         margin-right: 5px;
         border-radius: 3px;
-        width: $_icon_size;
-        height: $_icon_size;
+        width: $icon_size;
+        height: $icon_size;
```

Comment:
Name of variable `icon_size` should be written in all lowercase letters with hyphens instead of underscores